nexus/pkg/server/sender.go

```diff
@@ -246,6 +390,20 @@ func (s *Sender) sendRun(record *service.Record, run *service.RunRecord) {
 		s.logger.CaptureFatalAndPanic("sender received error", err)
 	}
 
+	if err := s.sendResume(s.RunRecord); err != nil {
+		result := &service.Result{
```

Comment:
add a log here on error... i know sendResume is very chatty already, but it feels like this is the place we should be logging also